### DISCUSSION TITLE
Drop-downs in Legacy

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#3186] [Support]
Hi, this is maybe a known issue, but it appears that drop-down menus in Legacy communities aren't working consistently. It appears to have something to do with whether the user is signed in or not. I've verified the following:
- If user is logged in, drop-downs seem to work consistently. I haven't experienced any issues on communities where I'm logged in.
- If user is logged out, drop-downs are mostly, but not universally, broken. E.g. [this community](https://genderhistory.pubpub.org/) still works for me when I'm logged out or using incognito, but [Commonplace](https://commonplace.knowledgefutures.org/32-recognition-and-rewards), [Academic Entrepreneurship](https://academicentrepreneurship.pubpub.org/), and [Open Pedagogy Portal](https://oen-openpedportal.pubpub.org/) do not.

**Comment by gabestein** [comment]
Thanks Kyle. This looks like a caching issue. I've soft-purged everything so that these should start to rollover. If you still encounter this in the next day or so, please let me know.

**Reply by kyledenlinger** [reply]
Yep, working as expected now. Thanks for the quick fix!
